dna/dna.json

```diff
@@ -277,6 +314,29 @@
         }
       ]
 
-    }
+    },{
+
+        "Name": "hclib",
+
+        "Description": "This is used as a lib",
+
+        "RibosomeType": "js",
+
+        "CodeFile": "hclib.js",
+
+        "Entries": [
+
+        ],
+
+        "Functions": [
+          {
+            "Name": "doGet",
```

Comment:
this function doesn't exist.  Not a big deal because you don't call it, but the function in the js is commented out and has a different name...  Also as a lib, I think the sharing shouldn't be "public", but rather "zome"